app/bundles/CampaignBundle/Form/Type/CampaignLeadSourceType.php

```diff
@@ -114,18 +104,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     }
 
     /**
-     * @param OptionsResolverInterface $resolver
+     * @param OptionsResolver $resolver
      */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setRequired(['source_choices']);
     }
 
     /**
      * @return string
      */
-    public function getName()
+    public function getBlockPrefix()
     {
-        return 'campaign_leadsource';
+        return self::class;
```

Comment:
In this case we should keep the original string. Because changing that would break some JS functionality.
```suggestion
        return 'campaign_leadsource';
```